array-filtering.js

```diff
@@ -0,0 +1,5 @@
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+var filtered = numbers.filter( function evenNumbers(number) {
```

Comment:
its not a problem but you don't have to name your callback function here if you dont want.

```
var filtered = numbers.filter( function(number){
  return number % 2 === 0;
});
```